src/api/ApiClient.js

```diff
@@ -1,18 +1,14 @@
-import base64 from 'base-64';
 import { encodeAsURI } from '../lib/util';
+import { apiFetch, Auth } from './apiFetch';
 
-const apiVersion = 'api/v1';
+export type Backend = 'dev' | 'google' | 'password';
```

Comment:
I like the refactor of `ApiClient.js`! Maybe we should rename it `apiClient.js` since it's not a class anymore?